lib/mint/http.ex

```diff
@@ -2,19 +2,19 @@ defmodule Mint.HTTP do
   @moduledoc """
   Processless HTTP connection data structure and functions.
 
-  Single interface for `Mint.HTTP1` and `Mint.HTTP2` with version negotiation support
-  and support for proxies.
+  Single interface for `Mint.HTTP1` and `Mint.HTTP2` with support for version
+  negotiation and proxies.
 
   ## Usage
 
   To establish a connection with a given server, use `connect/4`. This will
-  return an opaque data structure, `%Mint{}`, that represents the connection
-  to the server. To send a request, you can use `request/5`. Sending a request
+  return an opaque data structure, `%Mint.HTTP1{}` or `%Mint.HTTP2{}`, that represents
```

Comment:
It can also return a proxy, so I think we should not be specific on what is returned.

```suggestion
  return an opaque data structure that represents
```